### DISCUSSION TITLE
Sort icon idle state for all columns

### DISCUSSION THREAD

**Discussion by blessenabrahamnw18** [komarovalexander/ka-table#286] [Q&A]
Is it possible to show sort icon in an idle state for all columns?
I need to show the sort icon for all columns all the time;
Current behaviour: Sort icon is enabled after one click the column header.

**Comment by komarovalexander** [comment]
Hi @blessenabrahamnw18 there are different sorting modes: https://komarovalexander.github.io/ka-table/#/sorting-modes 
MultipleRemote is probably what you can use, but you will have to implement custom sorting then: https://komarovalexander.github.io/ka-table/#/sorting-custom-logic
